diff -Naru ezdxf-0.8.9/setup.py ezdxf-0.8.9-fixed/setup.py
--- ezdxf-0.8.9/setup.py	2018-11-28 14:00:32.000000000 +0100
+++ ezdxf-0.8.9-fixed/setup.py	2019-01-09 17:56:24.708267754 +0100
@@ -16,7 +16,7 @@
 else:
     # Python 2.7: package_data seems to be broken
     # added required data to MANIFEST.in
-    PACKAGE_DATA = dict()
+    PACKAGE_DATA = {str('ezdxf'): [str('templates/*.dxf'), str('pp/*.html'), str('pp/*.js'), str('pp/*.css'), ]}
 
 
 def read(fname, until=""):
